packages/shared/lib/core/error/classes/base-error.class.ts

```diff
@@ -1,15 +1,24 @@
+import { localize } from '@core/i18n'
 import { showAppNotification } from '@lib/notifications'
 
 import { IErrorParameters } from '../interfaces'
 
+/**
+ * The base error, containing logic for handling the different
+ * error parameters.
+ */
 export class BaseError extends Error {
     constructor(params?: IErrorParameters) {
-        const message = params?.message ?? ''
+        const message = (params?.localizeMessage ? localize(params?.message) : params?.message) ?? ''
 
-        if (params?.logError) {
+        if (params?.logToConsole) {
             console.error(message)
         }
 
+        if (params?.saveToErrorLog) {
```

Comment:
Is this functionality that will be added in a future release?

Reply:
No, you were just quick to review :sweat_smile: